source/parser.hera

```diff
@@ -8436,7 +8432,12 @@ Init
       if (varIds.length) {
         // get indent from first statement
         const indent = getIndent(statements[0])
-        statements.unshift([indent, "var ", varIds.join(", "), "\n"])
+        let delimiter = ";"
+        if (statements[0][1]?.parent?.root) {
+          delimiter = "\n"
```

Comment:
This seems dangerous to me. Automatic semicolon insertion may not apply. Why would the root be special? Oh, maybe you want `";\n"`.

Reply:
I just didn't want to update 57 more tests but I'll do it :)

Reply:
I think ASI would always apply since it is a var declaration.

Reply:
Ah, I think you're right: it's just initalized declarations that can have a problem with ASI. An uninitialized declaration should be safe.